# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/85_mm-2.7-rm.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur mm2.7.orig/src/Makefile mm2.7/src/Makefile
--- mm2.7.orig/src/Makefile	2003-05-15 00:19:21.000000000 +0200
+++ mm2.7/src/Makefile	2003-05-15 00:21:59.000000000 +0200
@@ -15,7 +15,7 @@
 # System V users may change the following to remove the "-s" and all will still work.
 LN = ln -s
 
-RM = rm
+RM = rm -f
 # To shut rm up on SGI machines (where symlinks have mode 0000)
 # RM = rm -f
 
